assets/components/src/web-preview/index.js

```diff
@@ -30,95 +30,152 @@ class WebPreview extends Component {
 	state = {
 		device: 'desktop',
 		loaded: false,
-		previewVisibility: false,
+		isPreviewVisible: false,
+		modalDOMElement: null,
 	};
 
+	/**
+	 * Create a div and append it to the body, so that the modal is on the top layer
+	 */
+	componentDidMount() {
+		const modalDOMElement = document.createElement( 'div' );
```

Comment:
Any thought that we should try to reuse the same `div` if it has already been created by a different instance of `WebPreview`? In certain situations we could end up with a large number of unused `div`s sitting around, which may be harmless but we should at least consider the possibility.

Reply:
Agreed :) Also that single identifiable div will be less cryptic in devtools:

![image](https://user-images.githubusercontent.com/7383192/72990498-87528900-3df0-11ea-986e-f029c85171bb.png)